.github/workflows/deployExpensifyHelp.yml

```
@@ -44,19 +43,18 @@ jobs:
           source: ./docs/
           destination: ./docs/_site
 
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@64bcae551a7b18bcb9a09042ddf1960979799187
+      - name: Deploy to Cloudflare Pages
+        uses: cloudflare/pages-action@f0a1cd58cd66095dee69bfa18fa5efd1dde93bca
+        id: deploy
         with:
-          path: ./docs/_site
-
-  # Deployment job
-  deploy:
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@af48cf94a42f2c634308b1c9dc0151830b6f190a
+          apiToken: ${{ secrets.CLOUDFLARE_PAGES_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          projectName: helpdot
+          directory: ./docs/_site
+
+      - name: Leave a comment on the PR
+        uses: actions-cool/maintain-one-comment@de04bd2a3750d86b324829a3ff34d47e48e16f4b
+        if: ${{ github.event_name == 'pull_request' }}
+        with:
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+          body: ${{ format('A preview of your ExpensifyHelp changes have been deployed to {0} ⚡️', steps.deploy.outputs.url) }}
```

```suggestion
          body: ${{ format('A preview of your ExpensifyHelp changes have been deployed to {0} ⚡️', steps.deploy.outputs.alias) }}
```

nab, this will show the pretty URL instead

---

Went with alias but it looks about the same